pkg/processor/build/builder.go

```diff
@@ -51,6 +51,12 @@ const (
 	functionConfigFileName = "function.yaml"
 )
 
+const (
+	stagingDirName    = "staging"
```

Comment:
If all below comments are considered, it might be that these names aren't necessarily needed as constants:
- b.stagingDir is accessible with GetStagingDir()
- download and decompress directories are only used locally within the same functions that create them